### ISSUE TITLE
PySpark: SparkTestingBaseTestCase.assertRDDEquals is not an assertion

### ISSUE THREAD

**Issue by dutchgecko** [holdenk/spark-testing-base#235]
I can understand that it would be desirable to have a function that returns a boolean for comparing two RDDs, but it seems that the name for this function has been chosen poorly, as it gives the impression that the user simply needs to call the function to insert an assertion in their test. The same problem occurs for the sister function `assertRDDEqualsWithOrder`.

This naming/behaviour is also inconsistent with the `sqltestcase` module, where the functions starting with "assert" *are* assertions.

It's probably too late to change the names or behaviour due to backwards incompatibility, but would it be possible to *clearly* document this discrepancy, at the very least in the python source?

**Comment by Gmousse** [comment]
Agreed ! One of my test was a false Positive one because someone simply called the assertRDDEquals.
